tools/dev/k3d/lib/grafana/grafana.libsonnet

```diff
@@ -0,0 +1,66 @@
+local k = import 'github.com/grafana/jsonnet-libs/ksonnet-util/kausal.libsonnet';
+local tanka = import 'github.com/grafana/jsonnet-libs/tanka-util/main.libsonnet';
+local spec = (import './spec.json').spec;
+local helm = tanka.helm.new(std.thisFile) {
+  template(name, chart, conf={})::
+    std.native('helmTemplate')(name, chart, conf { calledFrom: std.thisFile }),
+};
+{
+  _config+:: {
+    jaegerAgentName: error 'please provide $._config.jaegerAgentName',
+    jaegerAgentPort: 6831,
+    namespace: error 'plase provide $._config.namespace',
+    provisioningDir: '/etc/grafana/provisioning',
+    lokiUrl: error 'please provide $._config.lokiUrl',
+    grafana: {
+      datasources: [],
+    },
+  },
+
+  _images+:: {
+    grafana: {
+      repository: 'grafana/grafana',
+      tag: '8.2.5',
```

Comment:
Grafana's at 8.3.4 these days. Why not latest here?

Reply:
because copy/paste from somewhere? yes, not a good reason, updating!